pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -603,9 +612,9 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	}
 	// record that we are evicting the pod
 	m.recorder.AnnotatedEventf(pod, annotations, v1.EventTypeWarning, Reason, evictMsg)
-	// this is a blocking call and should only return when the pod and its containers are killed.
+	// this is a non-blocking call, it will return right away, semaphore (if non-nil) will be released once the pod is actually killed
```

Comment:
left-over
```suggestion
	// this is a non-blocking call, it will return right away. lock (if non-nil) will be released once the pod is actually killed.
```